SevenZip/SevenZipBase.cs

```diff
@@ -15,7 +15,8 @@ public abstract class SevenZipBase : MarshalByRefObject
     {
         private readonly bool _reportErrors;
         private readonly int _uniqueID;
-        private static readonly List<int> Identifiers = new List<int>();
+        private static readonly object _syncUniqueIds = new object();
```

Comment:
Once again preference, but renaming this something like `_uniqueIdLock` would make its purpose/use easier to identify.

Reply:
fixed